Added a way to play local video for sddm.

yehuohan opened the pull request:
- added a extra way to play local video;
It's a little slow to play the video via internet. so I add a way to play the local video for sddm.

- change the postition of login-faild-msg;
When the password is wrong, the login-failed-msg will be below password-box.

- added a button to clear the password;
With the clear-passwd-button, it's easier to re-type password.

- change the color of session and us-layout font
If the font color is white, the content of session and us-layout can't be see for the background color of combox is white too.

3ximus commented:
I changed a few things in your pull request:
 - I removed the code to generete the local playlists since it can be done with a single command (I added it to the README)
- I made the clear password button auto-hide when nothing is written in the password field, and overall works better when the LoginButton is hidden

I never gotten around to fix the rest of the stuff you did so thanks a lot :)
Feel free to tell me anything you didn't like.

yehuohan commented:
:+1: 
It's my pleasure.

Commit:
added a extra way to play local video;
change the postition of login-faild-msg;
added a button to clear the password;
change the color of session and us-layout font
